modules/jetstream-maxtext-deployment/templates/deployment.yaml.tftpl

```diff
@@ -0,0 +1,51 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: maxengine-server
+  namespace: default
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: maxengine-server
+  template:
+    metadata:
+      labels:
+        app: maxengine-server
+    spec:
+      nodeSelector:
+        cloud.google.com/gke-tpu-topology: 2x2
+        cloud.google.com/gke-tpu-accelerator: tpu-v5-lite-podslice
```

Comment:
Is this the only tpu accelerator that jetstream supports? Should this be parameterized so folks can run this on other tpu types?